### DISCUSSION TITLE
I added a new user to my cloud setup but not able to edit

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#11502] [Help] [answered]
To test out the newly added collaboration feature, I added one of my colleague as a new admin.
She can successfully sign in to my cloud and start editing the app without me being locked out. However, in the design tab, she sees:
![image](https://github.com/Budibase/budibase/assets/93232582/f1ad8fce-e5fe-4d9a-9341-fc04307a6fbc)
But I don't have this issue. Is anyone having the same problem?
(My cloud domain: apmedia.budibase.app)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @dannyshin76,

I have a few questions on the above if you could answer them it would be really helpful.
- Could you get the user send a screenshot of their **console log** as well as their **network requests** whenever they try to go to the design area?
- What browser are they using?
- What is the version of this browser?

Theres a few different reasons for a user to see the above screen. 
- One being they have a corrupted version of the budibase-client.js. **Can you get them to clear their cache and try again.** Additionally if they don't know how to clear their cache, they could try on a different browser to see if the error persists.
- It could also be a firewall blocking the connection to the CDN which creates the client library.

There are some other reasons but these would be the most common. 

We will be able to tell more when we get answers to the questions above.
Thanks in advance for the information.

**Reply by dannyshin76** [reply]
It was her computer's antivirus software marking the budibase cdn as malicious. I cleared the marking to get it right.
Thanks for the advise!

**Reply by ConorWebb96** [reply]
No problem. Happy that you got it sorted! 🤝
